pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -868,10 +868,17 @@ export class CreateVmAction extends React.Component {
         if (this.props.systemInfo.osInfoList == null)
             return null;
 
+        let testdata;
+        if (!this.props.systemInfo.osInfoList)
+            testdata = "disabledOsInfo";
+        else if (!this.state.virtInstallAvailable)
+            testdata = "disabledVirtInstall";
+        else if (this.state.downloadOSSupported === undefined)
+            testdata = "disabledDownloadOS";
         let createButton = (
-            <Button disabled={!(this.props.systemInfo.osInfoList &&
-                               (this.state.virtInstallAvailable != undefined ||
-                               (this.state.virtInstallAvailable && this.state.downloadOSSupported === undefined)))}
+            <Button disabled={!this.props.systemInfo.osInfoList || !this.state.virtInstallAvailable ||
+                              this.state.downloadOSSupported === undefined}
```

Comment:
So do you see any problem with:
```
<Button disabled={!(this.props.systemInfo.osINfoList && this.state.virtInstallAvailable !== undefined && this.state.downloadOSsupported !== undefined }
```

Reply:
what if `this.state.virtInstallAvailable` will be `false`? Button should be disabled but this condition would not disable the button.

Reply:
What if `this.state.virtInstallAvailable` will be `false`? In that case the button should be disabled but the condition would not disable it.

Reply:
Oh right - got confused again  - sorry for the chaos